Integrations/integration-CrowdStrikeHost.yml

```diff
@@ -0,0 +1,647 @@
+description: Crowdstrike IOCs and detections API
+detaileddescription: |-
+  Please provide the API id and key for the CrowsStrike Falcon Host API.
+  API Key Pairs can be requested by emailing support@crowdstrike.com.
+configuration:
+- display: Server URL (e.g. https://192.168.0.1)
+  name: url
+  defaultvalue: https://falconapi.crowdstrike.com/
+  type: 0
+  required: true
+- display: API ID
+  name: id
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: API Key
+  name: key
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Use system proxy settings
+  name: useproxy
+  defaultvalue: "true"
+  type: 8
+  required: false
+- display: Allow self-signed SSL certificates
+  name: insecure
+  defaultvalue: "false"
+  type: 8
+  required: false
+script:
+  script: |
+    var serverUrl = params.url;
```

Comment:
I'm used to have the `/` at the end always... don't think it matters much either way.